linkml_model/model/schema/meta.yaml

```diff
@@ -1333,6 +1360,15 @@ classes:
     close_mappings:
       - qb:HierarchicalCodeList
 
+  alias_definition:
```

Comment:
so our naming conventions are a bit odd. Only slot_definition and class_definition inherit from definition... yet schema_definition inherits from element, despite it's name...

How about we reserved Xdefinition for things with identifiers, we can simply call this structured_alias?

Reply:
done.